doc_src/source/roadmap.rst

```diff
@@ -2,30 +2,37 @@ Roadmap
 =======
 
 Version 0.1 of the package exposes the first optimization: multithreaded ufunc
-loops. Future versions of the package will extend the capabilites. Some of the
-proposed enhancements will require new API from NumPy.
+loops. 
+
+Future versions of the package will extend these capabilites to cover more of 
+the NumPy functionality. Some of these proposed enhancements will require new 
+API from NumPy.
```

Comment:
"a new API from Numpy?"

Reply:
I'm not sure. Something sounds funny about "enhancements will require new API from NumPy"

Reply:
I agree. But I didn't write it so I'm not sure how to handle that?